pom.xml

```diff
@@ -1192,7 +1200,12 @@ flexible messaging model and an intuitive client API.</description>
         <groupId>io.streamnative.oxia</groupId>
         <artifactId>oxia-client</artifactId>
         <version>${oxia.version}</version>
-        <classifier>shaded</classifier>
```

Comment:
Is there a problem with the shaded oxia-client? Would it be worth reporting directly to https://github.com/streamnative/oxia ?

Reply:
Sure, this shaded includes the `org.slf4j`.

Reply:
Isn't that something that shouldn't be shaded at all? would it be helpful to exclude it from shading?

Reply:
sorry, the Java client is at https://github.com/streamnative/oxia-java

Reply:
Issue: https://github.com/streamnative/oxia-java/issues/129

I think the oxia-client should remove the slf4j from the shaded jar.